oauthenticator/openshift.py

```diff
@@ -28,15 +28,25 @@ class OpenShiftOAuthenticator(OAuthenticator):
     scope = ['user:info']
```

Comment:
I believe that for this to work properly, we'd need to have `user:full` by default, because line 166 will fail if not

Reply:
@vpavlin are you sure? I am able to get back the groups a user is in even with the `user:info` scope in my test cluster and can persist them to `auth_state`. I thought that `user:full` was only for reserved for R/W

Reply:
Yeah, sorry, my bad, I guess I misunderstood the scope originally. `info` looks good